test/utils.ts

```diff
@@ -58,8 +58,15 @@ export function eventMock (options?: Partial<EventData>): EventData {
   return testEvent
 }
 
-export function blockMock (blockNumber: number, blockHash = '0x123'): BlockTransactionString {
+export function blockMock (blockNumber: number, blockHash = '0x123', options?: Partial<BlockTransactionString>): BlockTransactionString {
   const block = Substitute.for<BlockTransactionString>()
+  options = options || {}
+
+  for (const [key, value] of Object.entries(options)) {
+    // @ts-ignore
+    block[key].returns!(value)
+  }
+
```

Comment:
```suggestion

Object.entries(options).forEach([key, value] => {
    // @ts-ignore
    block[key].returns!(value)
  })

```

Reply:
it's just copy-paste. I also prefer to use built-in Array methods)